cmd/tink-controller-v1alpha2/main.go

```diff
@@ -0,0 +1,179 @@
+package main
```

Comment:
is v1alpha2 so different from a cmd/cli perspective that we have to break it out entirely? Could a flag be used in the existing tink-controller?

Reply:
I'm open to using a single binary. This temporary extra binary just felt simpler to me. There won't be a transition period where both APIs are supported so eventually `tink-controller-v1alpha2` will become `tink-controller`.